### ISSUE TITLE
Two-way audio not working through Tailscale

### ISSUE THREAD

**Issue by chwoa** [dermotduffy/frigate-hass-card#1278]
**Checklist:**

- [ x] I updated to the latest version available
- [ x] I cleared the cache of my browser

**Release with the issue:**

**Last working release (if known):**

**Browser and Operating System:**

Tried in Safari, Chrome and iOS app.
Using Tailscale and with HA behind a caddy reverse proxy.

**Description of problem:**
I use my Home Assistant behind a reverse proxy (caddy) which is connected to Tailscale for remote access.

Locally, I have two-way audio working perfectly through the card, but when connected from a different network (mobile) using Tailscale, the microphone button does not appear.

I am aware WebRTC would try to establish a P2P connection: WebRTC streams (even video only) through the go2rtc dashboard do not work through Tailscale, but I believe that is expected.

Video and audio seem to work perfectly fine through WebRTC (both go2rtc and webrtc card as providers) when using Tailscale, though.
I really don't understand how :) , and I am not familiar with your code at all, but maye the card uses Frigate/HA Ingress to direct the traffic?

Could something similar be done for the microphone too? Or this is some limitation that requires further configuration?

This seems somewhat similar to https://github.com/dermotduffy/frigate-hass-card/issues/1255 (they mention Zerotier there), but I have double checked everything is using HTTPS, plus it does work locally with the same app/browser.

**Frigate card diagnostic information**

```json

```

**Javascript errors shown in the web inspector (if applicable):**

```

```

**Additional information:**

**Comment by dermotduffy** [comment]
I think this is unlikely to be an issue related to the card. All network traffic from the card to go2rtc in your Frigate installation is proxied via the Frigate integration which runs as part of your HA process. That behavior is consistent whether you are local or not. If it doesn't work when you are remote, but everything else in Home Assistant appears to work okay remotely, there must be something odd going on with your reverse proxy and/or you are not using `https` (causing your browser to not offer you audio).

TL;DR: There's no reason this shouldn't just work.

**Comment by chwoa** [comment]
Thank you for your answer!
Adding the machine where the Home Assistant lives to the Tailnet works around the issue, and I can see in the logs that indeed a P2P connection is set between its Tailscale IP and the client's when I start the two way audio in the card. So it seems that Frigate does proxy WebRTC and audio, but not the microphone, which stills requires raw WebRTC communication. I might ask in Frigate :)
